packages/core/upload/admin/src/pages/App/components/ImageAssetCard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  Card,
+  CardAction,
+  CardAsset,
+  CardBadge,
+  CardBody,
+  CardCheckbox,
+  CardContent,
+  CardHeader,
+  CardTitle,
+  CardSubtitle,
+} from '@strapi/parts/Card';
+import { IconButton } from '@strapi/parts/IconButton';
+import EditIcon from '@strapi/icons/EditIcon';
+import { useIntl } from 'react-intl';
+import { getTrad } from '../../../utils';
+
+const Extension = styled.span`
+  text-transform: uppercase;
+`;
+
+export const ImageAssetCard = ({ name, extension, height, width, thumbnail }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Card>
+      <CardHeader>
+        <CardCheckbox value />
+        <CardAction position="end">
+          <IconButton
+            label={formatMessage({ id: getTrad('control-card.edit'), defaultMessage: 'Edit' })}
+            icon={<EditIcon />}
+          />
+        </CardAction>
+        <CardAsset src={`http://localhost:1337${thumbnail}`} />
```

Comment:
You can use the `prefixFileUrlWithBackendUrl` util from the helper plugin instead of hard coding the backend url.

Also do you mind moving this util from the `old/utils` folder to the new utils one and a very small story to explain what it does?